iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/sql/ast/CreateView.java

```diff
@@ -92,4 +98,16 @@ public String toString() {
         .add("restrict", restrict)
         .toString();
   }
+
+  @Override
+  public long ramBytesUsed() {
+    long size = super.ramBytesUsed();
+    // super.ramBytesUsed() includes CreateTable's INSTANCE_SIZE, but we need CreateView's
+    size -= RamUsageEstimator.shallowSizeOfInstance(CreateTable.class);
```

Comment:
no need to calculate this each time, you should define a method without CreateTable instance size in super class.